src/test/java/org/jenkinsci/plugins/DependencyCheck/model/ReportParserTest.java

```diff
@@ -1,17 +1,81 @@
 package org.jenkinsci.plugins.DependencyCheck.model;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.fail;
 
 import java.lang.reflect.InvocationTargetException;
+import java.util.List;
 
 import org.junit.Test;
 
 public class ReportParserTest {
-    @Test(expected = InvocationTargetException.class)
-    public void testRejectsExternalEntities() throws Exception
-    {
-        new ReportParser(1).parse(getClass().getResourceAsStream("dependency-check-report-external-entities.xml"));
 
-        fail("Should have rejected input with external entities");
-    }
+	@Test(expected = InvocationTargetException.class)
```

Comment:
Inconsistent indents (4 vs 8 spaces, sadly ignored in checkstyle).

Reply:
I will correct that....